### DISCUSSION TITLE
paddleocr如何支持返回单个字以及单个字的坐标？

### DISCUSSION THREAD

**Discussion by anyhopex** [PaddlePaddle/PaddleOCR#13857] [Q&A] [answered]
Name: paddleocr
Version: 2.8.1

**Comment by GreatV** [comment]
- #10377

**Reply by anyhopex** [reply, marked as answer]
设置：return_word_box = True时，返回：
[[[26.0, 37.0], [304.0, 37.0], [304.0, 73.0], [26.0, 73.0]], ('纯臻营养护发素', 0.9946897625923157, [46.085826210826205, [['纯', '臻', '营', '养', '护', '发', '素']], [[3, 10, 16, 23, 30, 36, 43]], ['cn']])] 

请问这个返回怎么使用，有没有说明，怎么根据这个返回获取单字具体坐标呢？

**Comment by 759325100** [comment]
使用 https://github.com/PaddlePaddle/PaddleOCR/issues/10377 的方案并不能很好的解决这个问题。如场景是影印版本的pdf，这个效果定位会更加不准确。下图是我将CTC返回的col位置索引  * cellWh (cellWh = box-width / col-len）得到的每个字符的初识坐标，不管在中文、符号或数字的情况下，都并未呈现出某种一致的规律，请问CTC返回的这个所谓的位置，底层到底是什么样的逻辑～ 
![111](https://github.com/user-attachments/assets/8c8d5e1e-f16b-4448-9488-41c06dff0852)

**Reply by 759325100** [reply]
@GreatV  期待您的答复。

**Comment by SWHL** [comment]
可以试试我们在RapidOCR中集成的PaddleOCR单字坐标：https://github.com/RapidAI/RapidOCR/releases/tag/v1.4.0
